utils/ratelimits.py

```diff
@@ -65,6 +69,20 @@ def wrapper(*args, **kwargs):
                                         'X-RateLimit-Remaining': max_usage - usage - 1,
                                         'X-RateLimit-Reset': cache.expires_on(key['id'])}))
             else:
+                try:
+                    ratelimit_reached = key['ratelimit_reached']
+                except KeyError:
+                    ratelimit_reached = 0
+                ratelimit_reached += 1
+                print(ratelimit_reached)
+                r.table('keys').get(auth).update({"ratelimit_reached": ratelimit_reached}).run(get_db())
+                if ratelimit_reached % 5 == 0 and 'webhook_url' in config:
+                    requests.post(config['webhook_url'], headers={'content-type': 'application/json'},
+                                  data=json.dumps({"embeds": [{
+                                      "title": f"Application '{key['name']}' ratelimited 5 times!",
```

Comment:
Is it worth showing the total amount of times they've hit ratelimit rather than 5 each time?

Reply:
Is does that in the embed, but the embed fires every 5 times a ratelimit is reached, so if someone is spamming you can :b::a:n

Reply:
o ok